must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -205,14 +205,32 @@ for ns in $namespaces; do
 
     oc delete -f pod_helper.yaml
 
+    crash_collection(){
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        while [ "$(oc get pods -n openshift-storage | grep "${node//./}-debug" | awk '{print $2}')" != "1/1" ] ; do
```

Comment:
Do these pods have labels we could use?

So we could do something like:

```bash
[[ "$(oc get pods -n openshift-storage -l app=DEBUG_LABEL -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]
```

Reply:
I agree, this would be cleaner and more reliable.

Reply:
there can be multiple debug pods how to know which one is for which.

Reply:
I checked no labels

Reply:
Ah well! I can probably live with this for now.